ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -145,6 +149,7 @@ pipeline {
             sh "cd ${TESTDIR}; K8S_VERSION=1.8 vagrant destroy -f || true"
```

Comment:
Does this mean that we are only destroying the VMs after the entire pipeline has run? I.e., the K8S 1.8 and 1.9 VMs are still running at the start of the stage for K8s 1.10 and K8s 1.12?

Reply:
https://github.com/cilium/cilium/blob/0c9728feab4f3354634e8db07512ac4b72a66137/test/test_suite_test.go#L245-L262

Reply:
Great, thanks @eloycoto !